WIP-extension.md

```diff
@@ -1,7 +1,33 @@
 Q:
 
-- [ ] Is FE aware of current crowdfunding phase?
+- [x] Is FE aware of current crowdfunding phase? _Yes_
 
+Flow to determine whether a prolong `packageOption` is returned.
+
+Input:
+- A membership
+- A prolong packageOption, to check against
+
+a)  Does user own membership, or pledge membership (ABO_GIVE)
+    - If false, end and don't return package option
+b)  Is membership active
+    - If false, end
+c)  Is current membershipPeriod last period
+    - If false, end
```

Comment:
c is optional I think. It's not mandatory to avoid extensions of extensions.

Reply:
Feels like this would have implications on finance. Added to question list.